packages/react-instantsearch/src/widgets/__tests__/SearchBox.test.tsx

```diff
@@ -151,115 +24,6 @@ describe('SearchBox', () => {
     });
   });
```

Comment:
This could be moved in the common test suite by taking into account case adjustments for `autoFocus` in IS.js and Vue, in their package's common test implementation. Wdyt?

Reply:
Problem is they are handled differently, in IS.js it adds the `autofocus` attribute but for some reason jsdom does not focus it so we have to test the attribute rather than `toHaveFocus`.
For React, [it doesn't get added to the list of attributes](https://github.com/facebook/react/blob/34de2986dfeddda31cb76d298c6d59b271d976fa/packages/react-dom-bindings/src/client/ReactDOMComponent.js#L636-L641) but it gets polyfilled so here we can only test with `toHaveFocus` 🫠

Reply:
I could do something like:
```js
try {
  expect(screen.getByRole('searchbox').toHaveAttribute('autofocus', 'true');
catch {
  expect(screen.getByRole('searchbox').toHaveFocus();
}
```